standalone/README.md

```diff
@@ -113,50 +182,28 @@ This will provision the necessary servers for an instance of simple-server on di
 
 - Update the vault by running:
     ```bash
-    cat terraform.tfvars | ansible-vault encrypt --vault-id ../../password_file --output terraform.tfvars.vault
+    cat terraform.tfvars | ansible-vault encrypt --vault-id ../../vault_password --output terraform.tfvars.vault
     ```
   Check in the updated vault.
 
-## Helpful Commands
+## Troubleshooting
 
-### Editing vault files
+Depending on your system, you may run into the following known issues:
 
-There are other vault files that are checked into this repository that do not have a corresponding local decrypted version
-for development. You can view or edit the contents of these vault files directly by running:
+### Errors with cloudalchemy.node-exporter
 
-```bash
-ansible-vault view --vault-id ../../password_file roles/load-balancing/vars/ssl-vault.yml
-ansible-vault edit --vault-id ../../password_file roles/load-balancing/vars/ssl-vault.yml
-```
+If you see output like this, it's likely due to [this issue](https://github.com/cloudalchemy/ansible-node-exporter/issues/54).
 
-### Making a deploy
-```bash
-make deploy hosts=icmr/playground
 ```
-This deploys simple-server/master on hosts.
-
-### Updating ssh keys
-Add keys to `ansible/roles/ssh/` under the appropriate environment.
-```bash
-make update-ssh-keys hosts=icmr/playground
+TASK [cloudalchemy.node-exporter : Get checksum list from github] *******************************************************************************************************************************************************
+objc[5848]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called.
+objc[5848]: +[__NSCFConstantString initialize] may have been in progress in another thread when fork() was called. We cannot safely call it or ignore it in the fork() child process. Crashing instead. Set a breakpoint on objc_initializeAfterForkError to debug.
+ERROR! A worker was found in a dead state
 ```
-Note that this clears any old keys present on the servers.
 
-### Updating app config
-The app's .env file sits in `ansible/roles/deploy/templates/.env.j2`.
-Variables are sourced from `ansible/roles/deploy/templates/vars`
-```bash
-make update-app-config hosts=icmr/playground
-```
 
-### Restarting passenger
-```bash
-make restart-passenger hosts=icmr/playground
+#### Run this to fix:
 ```
-Note that this restarts passenger on all servers.
-
-### Restarting sidekiq
-```bash
-make restart-sidekiq hosts=icmr/playground
+export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES
```

Comment:
But this is now baked into the script already. Do we need this troubleshooting section?

Reply:
This is in the `Makefile`, still handy to have it documents in case one runs ansible commands.